lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -9639,6 +9639,86 @@ namespace Js
         return superCtor;
     }
 
+    Var JavascriptOperators::OP_ImportCall(Var specifier, ScriptContext* scriptContext)
+    {
+        ModuleRecordBase *moduleRecordBase = nullptr;
+        SourceTextModuleRecord *moduleRecord = nullptr;
+
+        JavascriptFunction* pfuncCaller;
+        if (JavascriptStackWalker::GetCaller(&pfuncCaller, scriptContext) && pfuncCaller && pfuncCaller->IsScriptFunction())
```

Comment:
Updated interpreter and lowerer code to pass in function and removed stack walker call.

---
In reply to: [114930380](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r114930380) [](ancestors = 114930380)